src/pages/Home/index.js

```diff
@@ -51,27 +51,44 @@ const Home = () => {
   return (
     <div className="home-container">
       <div className="word-container">
-        {usersAttempts.map((attempt, round) => (
-          <div className="word" key={`word-attempt-${round}`}>
-            <>
-              {attempt.map((c, index) => (
-                <LetterInput
-                  key={`${c}-${round}-${index}`}
-                  value={usersAttempts[round][index]}
-                  style={{
-                    backgroundColor:
-                      round !== currentRound || gameEnded
-                        ? LETTER_STATUS[roundsResults[round][index]].color
-                        : LETTER_STATUS.nothing.color,
-                  }}
-                  isSelected={index === letterIndex && round === currentRound}
-                  onClick={() => setLetterIndex(index)}
-                  disabled={round !== currentRound || gameEnded}
-                />
-              ))}
-            </>
-          </div>
-        ))}
+        {usersAttempts.map((attempt, round) => {
+          const attemptedWord = attempt.join('');
+          const wordAriaLabel = !!attemptedWord.length ? attemptedWord : t('ariaLabels.empty');
```

Comment:
i think this should work

```suggestion
          const wordAriaLabel = attemptedWord ?? t('ariaLabels.empty');
```

Reply:
thanks, I'll do the same as JP's suggestion: 

```suggestion
          const wordAriaLabel =  attemptedWord || t('ariaLabels.empty');
```

Reply:
cool!